packages/myst-transforms/src/blockquote.spec.ts

```diff
@@ -17,7 +78,24 @@ describe('Test blockquoteTransform', () => {
           u('blockquote', [
             u('paragraph', [u('text', 'We know what we are, but know not what we may be.')]),
           ]),
-          u('caption', [u('paragraph', [u('text', 'Hamlet act 4, Scene 5')])]),
+          u('caption', [u('paragraph', [u('strong', [u('text', 'Hamlet act 4, Scene 5')])])]),
+        ]),
+      ]),
+    );
+  });
+  test('blockquote with empty attribution is unchanged', async () => {
+    const mdast = u('root', [
+      u('blockquote', [
+        u('paragraph', [u('text', 'We know what we are, but know not what we may be.')]),
+        u('paragraph', [u('text', '-- ')]),
```

Comment:
Does this also work?

```suggestion
        u('paragraph', [u('text', '-- '), u('text', '')]),
```

Reply:
That _will_ leave an empty attribution. If we don't want this (and it's possible to produce via a parse), then should we e.g. check whether the `toString` (or whatever function we have) is non-empty?

Reply:
I think that is fine and we don't have to catch that. I am not sure how that would be written in markdown anyways.